core/extensions.js

```diff
@@ -102,10 +103,11 @@ const registerMutator = function(name, mixinObj, opt_helperFn, opt_blockList) {
   // Sanity checks passed.
   register(name, function() {
     if (hasMutatorDialog) {
-      if (!Blockly.Mutator) {
+      const Mutator = goog.module.get('Blockly.Mutator');
+      if (!Mutator) {
         throw Error(errorPrefix + 'Missing require for Blockly.Mutator');
       }
-      this.setMutator(new Blockly.Mutator(opt_blockList || []));
+      this.setMutator(new Mutator(opt_blockList || []));
```

Comment:
N.B.: this will shadow an import of `Blockly.Mutator` aliased as `Mutator`, but since they refer to the same thing I think that's quite OK.